pkg/prowgen/prowgen_images_discovery.go

```diff
@@ -19,7 +20,13 @@ const (
 	srcImage = "src"
 )
 
-var registryRegex = regexp.MustCompile(`registry\.(|svc\.)ci\.openshift\.org/\S+`)
+var (
+	registryRegex             = regexp.MustCompile(`registry\.(|svc\.)ci\.openshift\.org/\S+`)
+	defaultDockerfileIncludes = []string{
+		"openshift/ci-operator/knative-images.*",
+		"openshift/ci-operator/knative-test-images.*",
+	}
```

Comment:
To avoid breaking EKB we need to add a new pattern `openshift/ci-operator/static-images.*` for repositories that want to define Dockerfiles that are not generated by `hack/cmd/generate` (like EKB)

Reply:
Noticed it here https://github.com/openshift/release/pull/45546/files#r1393922350

Reply:
PR https://github.com/openshift-knative/hack/pull/91